lib/plugins/verify/index.js

```diff
@@ -53,6 +53,15 @@ async function verify(server, options) {
           )
         })
 
+        // The apkDigestSha256 comparison check exists to ensure that the request has come from a known
+        // version of the App. This check compares the hash (digest) of the APK file on the phone with the
+        // APK file the system is configured to allow.
+        //
+        // Note that this check assumes that there is exactly 1 valid APK version in use at any single time.
+        // This will prevent multiple versions of the App from being usable at any given time.
+        //
+        // If you'd rather support multiple versions of the app at the same time then the APK check can be
+        // turned off by setting the config DEVICE_CHECK_PACKAGE_DIGEST to null.
```

Comment:
Fixes #14: Documents why this check exists, what it means, and how to disable it.